.github/setup-linux.sh

```diff
@@ -58,6 +58,11 @@ if [ "$1" == "ossl3" -o "$2" == "ossl3" ]; then
 	./.github/setup-openssl.sh
 fi
 
+# install libressl if needed
+if [ "$1" == "libressl" -o "$2" == "libressl" ]; then
```

Comment:
I think it's better to provoke an error if libressl and ossl3 were given. otherwise it's possible to pass both options and the script would *silently* ignore libressl.